submissions/task_html-css-popup/index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Hooli-Style Popup</title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+    </head>
+    <body>
+        <header class="header">
+            <nav class="nav">
+                <div class="wrapper">
+                        <nav class="menu header__menu">
+                            <ul class="menu__list">
+                                <li class="menu__item"><a href="#" class="menu__link">Watchlist</a></li>
+                                <li class="menu__item"><a href="#" class="menu__link">Subscribe</a></li>
+                                <li class="menu__item"><a href="#" class="menu__link">Sign In</a></li>
+                                <li class="menu__item">
+                                    <input type="checkbox" id="burgerBtn" class="menu__burger-checkbox">

```

Comment:
Now when you navigate with keyboard, menu is not opens on focus on menu icon, it is not good UX 
Please, try to move input in label, and add :focus state to label

Reply:
Done. Unfortunately, I cannot change the the files index.html & style.css in the folder _submissions_ -> https://github.com/julia-online/frontend-2022-homeworks/submissions/task_html-css-popup/index.html
I don't know how to proceed with this - could you please help? Thx

Reply:
Thank you, Hanna! 🤗 The popup has undergone a complete restyling.
Hope I've made a few mistakes this time... 😅